subtitles/zh-CN/59_data-processing-for-translation.srt

```diff
@@ -255,18 +255,18 @@ to do this all automatically.
 
 52
 00:02:10,500 --> 00:02:13,800
-然后你可以将它与你的数据集一起传递给培训师
+之后你可以在你的 Keras 模型上使用 model.fit () 之前
 You can then pass it to the Trainer with your datasets
 
 53
 00:02:13,800 --> 00:02:15,960
-或者在 to_tf_dataset 方法中使用它
+将它与你的数据集一起传递给 Trainer
 or use it in the to_tf_dataset method
 
 54
 00:02:15,960 --> 00:02:18,560
-在你的（模糊的）模型上使用 model.fit () 之前。
-before using model.fit () on your (indistinct) model.
+或者在 to_tf_dataset 方法中使用它。
+before using model.fit () on your Keras model.

```

Comment:
I had a listen and agree this looks right! I've also fixed the English source here: https://github.com/huggingface/course/pull/411

Reply:
great!